Marlin/src/gcode/bedlevel/abl/G29.cpp

```diff
@@ -109,6 +109,10 @@ class G29_State {
   #elif ENABLED(AUTO_BED_LEVELING_3POINT)
     static constexpr grid_count_t abl_points = 3;
   #elif ABL_USES_GRID
+    /**
+     * in case of AUTO_BED_LEVELING_BILINEAR, this has the meaning of abl_max_points, the actual number, that is currently being probed can be lower
+     * If PROBE_MANUALLY is mutually exclusive with BILINEAR, abl_points is not needed when BILINEAR is active
+     */
     static constexpr grid_count_t abl_points = GRID_MAX_POINTS;
   #endif
```

Comment:
Rather than adding more annoying commits I'll just add annoying comments. Here you can use…
```cpp
#if ANY(AUTO_BED_LEVELING_LINEAR, VARIABLE_GRID_POINTS)
  grid_count_t abl_points;
#elif ENABLED(AUTO_BED_LEVELING_3POINT)
  static constexpr grid_count_t abl_points = 3;
#elif ABL_USES_GRID
  static constexpr grid_count_t abl_points = GRID_MAX_POINTS;
#endif
```